addons/indicators/stringtable.xml

```diff
@@ -89,72 +100,84 @@
                 <Japanese>通常</Japanese>
                 <Polish>Standard</Polish>
                 <Portuguese>Padrão</Portuguese>
+                <Russian>Стандарт</Russian>
             </Key>
             <Key ID="STR_dui_indicators_indicator_square">
                 <English>Square</English>
                 <German>Rechteck</German>
                 <Polish>Kwadrat</Polish>
                 <Portuguese>Quadrado</Portuguese>
                 <Japanese>四角</Japanese>
+                <Russian>Квадрат</Russian>
             </Key>
             <Key ID="STR_dui_indicators_indicator_diamond">
                 <English>Diamond</English>
                 <German>Diamant</German>
                 <Polish>Diament</Polish>
                 <Portuguese>Diamante</Portuguese>
                 <Japanese>ダイアモンド</Japanese>
+                <Russian>Ромб</Russian>
             </Key>
             <Key ID="STR_dui_indicators_indicator_circle">
                 <English>Circle</English>
                 <German>Kreis</German>
                 <Polish>Kółko</Polish>
                 <Portuguese>Círculo</Portuguese>
                 <Japanese>円型</Japanese>
+                <Russian>Круг</Russian>
             </Key>
             <!-- Icon Toggle -->
             <Key ID="STR_dui_cat_icons">
                 <English>Icons</English>
                 <German>Icons</German>
                 <Japanese>アイコン</Japanese>
                 <Portuguese>Ícones</Portuguese>
+                <Russian>Иконки</Russian>
             </Key>
             <Key ID="STR_dui_indicators_icon_leader">
                 <English>Show Leader icon</English>
                 <German>Gruppenführer anzeigen</German>
                 <Japanese>リーダー アイコンを表示</Japanese>
                 <Portuguese>Mostrar ícone de Líder</Portuguese>
+                <Russian>Показать иконку командира</Russian>
             </Key>
             <Key ID="STR_dui_indicators_icon_buddy">
                 <English>Show Buddy icon</English>
                 <German>Buddy anzeigen</German>
                 <Japanese>バディ アイコンを表示</Japanese>
                 <Portuguese>Mostrar ícone de Parceiro</Portuguese>
+                <Russian>Показать Buggy иконку</Russian>
```

Comment:
I am not speaking russian, but I don't think "Buggy" is really correct

Reply:
i dont know what is "buggy"))) I leave some words or sentences to see in the mod itself what they influence or what they mean. then I'll finish the translation, sometimes it is necessary not only to simply translate, but also to explain literally normally. not all players can understand the subtleties of your ideas) and here the main thing is to convey the meaning of the phrase, and not the phrase itself verbatim. I translated 10,000 lines of xeno domination)) I can handle it with your mod) just add corrections, and as updates will be added to the translation and edit. I think that's okay.

Reply:
The reason I added that comment was because the original word is "buddy" not "buggy". I do understand that leaving the original words untranslated has a reason. I am doing the same thing when translating to German.

Reply:
sorry this is just a typo